apache/rocketmq/v1/service.proto

```diff
@@ -437,6 +436,10 @@ service MessagingService {
 
   rpc MultiplexingCall(MultiplexingRequest) returns (MultiplexingResponse) {}
 
+  rpc SendThreadStackResult(SendThreadStackRequest) returns (SendThreadStackResponse) {}
```

Comment:
Clients actually report/upload thread stack traces to servers. Would "ReportThreadStackTrace" be more accurate?